lib/corsac_jwt.dart

```diff
@@ -185,8 +185,12 @@ class JWTBuilder {
   }
 
   /// Token audience (standard `aud` claim).
-  void set audience(String audience) {
-    _claims['aud'] = audience;
+  void set audience(dynamic audience) {
+    if (audience is String) {
+      _claims['aud'] = [audience];
+    } else if (audience is List<String>) {
+      _claims['aud'] = audience;
+    }
```

Comment:
Similarly here, if someone wants to set `aud` as a single string we shouldn't force it to be an array.